connector/manifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.0.0' name='MongoSQL' version='18.1' min-version-tableau='2020.4'>
+<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.1.0' name='MongoSQL' version='18.1' min-version-tableau='2021.1'>
```

Comment:
Is _min-version-tableau='2021.1'_ not going to be too restrictive and cut us from potential users who are using a slightly old version of Tableau?
Was _2020.4_ looking to old?

Reply:
Good question, the 2021.1 version allows for the inferred connection resolver, which is used in the connector however may not be necessary.  I'll test out with setting the resolver and checking again if there is a benefit.

Reply:
I added back the `connection-normalizer` and tested with 2020.4 and it worked fine.  2020.3 is the first version that uses the Connection Dialogs V2, but that version is no longer available due to the log4j vulnerability so `2020.4` is a good min version.